astronomer/providers/core/operators/external_trigger_dagrun.py

```diff
@@ -0,0 +1,417 @@
+        hook = HttpHook(method="GET", http_conn_id=http_conn_id)
+
+        return hook.url_from_endpoint(endpoint=endpoint)
+
+
+class ExternalDeploymentTriggerDagRunOperator(BaseOperator):
+    """
+        External Deployment Trigger Dag Run Operator Make HTTP call to trigger a Dag Run and poll for the response state of externally
+        deployed DAG run to complete. The host should be external deployment url, header must contain access token
+
+        .. seealso::
+            - `Retrieve an access token and Deployment URL <https://docs.astronomer.io/astro/airflow-api#step-1-retrieve-an-access-token-and-deployment-url.>`_
+
+        :param http_conn_id: The HTTP Connection ID to run the operator against
+        :param headers: The HTTP headers to be added to the GET request
+        :param extra_options: Extra options for the 'requests' library, see the
+            'requests' documentation (options to modify timeout, ssl, etc.)
+        :param tcp_keep_alive: Enable TCP Keep Alive for the connection.
+        :param tcp_keep_alive_idle: The TCP Keep Alive Idle parameter (corresponds to ``socket.TCP_KEEPIDLE``).
+        :param tcp_keep_alive_count: The TCP Keep Alive count parameter (corresponds to ``socket.TCP_KEEPCNT``)
+        :param tcp_keep_alive_interval: The TCP Keep Alive interval parameter (corresponds to
+            ``socket.TCP_KEEPINTVL``)
+        :param trigger_dag_id: The dag_id to trigger (templated).
+        :param trigger_run_id: The run ID to use for the triggered DAG run (templated).
+            If not provided, a run ID will be automatically generated.
+        :param conf: Configuration for the DAG run (templated).
+        :param execution_date: Execution date for the dag (templated).
+        :param reset_dag_run: Whether clear existing dag run if already exists.
+            This is useful when backfill or rerun an existing dag run.
+            This only resets (not recreates) the dag run.
+            Dag run conf is immutable and will not be reset on rerun of an existing dag run.
+            When reset_dag_run=False and dag run exists, DagRunAlreadyExists will be raised.
+            When reset_dag_run=True and dag run exists, existing dag run will be cleared to rerun.
+        :param wait_for_completion: Whether wait for dag run completion. (default: False)
+        :param poke_interval: Poke interval to check dag run status when wait_for_completion=True.
+            (default: 60)
+        :param allowed_states: List of allowed states, default is ``['success']``.
+        :param failed_states: List of failed or dis-allowed states, default is ``None``.
+
+    Changed in version 2.1.3: 'queued' is added as a possible value.
+    """
+
+    template_fields: Sequence[str] = (
+        "headers",
+        "trigger_dag_id",
+        "trigger_run_id",
+        "execution_date",
+        "conf",
+    )
+    template_fields_renderers = {"headers": "json", "data": "py", "conf": "py"}
+    ui_color = "#7352ba"
```

Comment:
Will it be possible to add a comment here regarding what colour this hash code corresponds to :)?
Looks like `Blue Marguerite` https://colors.dopely.top/color-pedia/7352ba

Also curious to understand if there was some significance behind choosing this colour. Is it that we use it in some other places too?

Reply:
removed it completely. no need to be fancy with custom UI colors, the default one works.